lib/puppet-lint/plugins/check_parameter_documentation.rb

```diff
@@ -45,8 +50,17 @@ def check
         end
       rescue StopIteration; end
 
+      # warn about duplicates
+      doc_params_duplicates.each do |p|
+         notify :warning, {
+           :message => "Duplicate parameter documentation for #{p[:value]}",
+           :line    => p[:token].line,
+           :column  => p[:token].column + p[:token].value.match(/\A\s*(@param\s*)?/)[0].length + 1
```

Comment:
The `+ 1` part is technically not entirely correct I think. It should be the length of the token itself, right? So if you use `//`, should it be 2?  Perhaps you can use `p[:token].type.length`.

Also wondering if it makes sense to have a local variable `token`. It's not that much shorter though.

Reply:
I never knew about http://puppet-lint.com/checks/slash_comments/.  Dunno when they last worked.  Certainly not anymore though.  puppet-lint will also already warn about multiple line comments (which *do* still work but were removed from the puppet 6 docs).  puppet-strings won't process tags in multi-line comments, so I think hardcoding '1' isn't all that bad.

Reply:
In that case I'd stop parsing them (see line 11)

Reply:
`p[:token].type` is `COMMENT`, so length is 7.  Since `:SLASH_COMMENT` doesn't seem to be valid puppet for a long time and we have zero tests for `:MLCOMMENT` and core puppet-lint already warns about that, perhaps we should remove support for them in a separate PR?

Reply:
ok.  May as well remove support for them in this PR.

Reply:
PR updated